src/PlusDataCollection/CMakeLists.txt

```diff
@@ -1824,6 +1825,61 @@ IF(PLUS_USE_OpenCV_VIDEO)
     )
 ENDIF()
 
+#---------------------------------------------------------------------------
+# InfraredSeekCam
+IF(PLUS_USE_INFRARED_SEEK_CAM)
+  # Check OpenCV 
+  IF(NOT PLUS_USE_OpenCV)
+    MESSAGE("PLUS_USE_INFRARED_SEEK_CAM requires PLUS_USE_OpenCV option enabled, enabling.")
+    SET(PLUS_USE_OpenCV ON CACHE BOOL "Use OpenCV for Thermal Infrared Seek Camera." FORCE)
+  ENDIF()
+
+  # Check if the LibUSB is installed
+  if (NOT LIBUSB_INCLUDE_DIR)
```

Comment:
This finding code should go into a Find module, and loaded via FIND_PACKAGE(libusb)

Reply:
Someone has already done it here:
[https://github.com/IntelRealSense/librealsense/blob/master/CMake/FindLibUSB.cmake](https://github.com/IntelRealSense/librealsense/blob/master/CMake/FindLibUSB.cmake)

Reply:
Ok, I will do that. Thank you.